src/main/java/it/unisa/Amigo/gruppo/controller/GruppoController.java

```diff
@@ -27,10 +31,11 @@
      * @return il path della pagina su cui eseguire il redirect
      */
     @GetMapping("/gruppi/{id}")
-    public String visualizzaMembriSupergruppo(Model model, @PathVariable(name = "id")int idSupergruppo){
-        Persona personaLoggata = gruppoService.visualizzaPersonaLoggata();
-        prepareCandidateList(idSupergruppo,model,gruppoService.visualizzaListaMembriSupergruppo(idSupergruppo));
-        model.addAttribute("isResponsabile", gruppoService.isResponsabile(personaLoggata.getId(),idSupergruppo));
+    public String findAllMembriInSupergruppo(Model model, @PathVariable(name = "id")int idSupergruppo){
+        Persona personaLoggata = gruppoService.getAuthenticatedUser();
+        model.addAttribute("isCapogruppo", gruppoService.isResponsabile(personaLoggata.getId(), idSupergruppo));
```

Comment:
```suggestion
        model.addAttribute("isResponsabile", gruppoService.isResponsabile(personaLoggata.getId(), idSupergruppo));
```
Così l'attributo è consistente con il nome del metodo